src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommandValidator.cs

```diff
@@ -269,7 +269,7 @@ internal sealed class CreateDialogDialogActivityDtoValidator : AbstractValidator
             .NotEqual(x => x.Id)
             .When(x => x.RelatedActivityId.HasValue);
         RuleFor(x => x.PerformedBy)
-            .SetValidator(localizationsValidator);
+            .MaximumLength(Constants.DefaultMaxStringLength);
```

Comment:
Var det riktig å ikke bruke DefaultMaxStringLength, men heller 100? 🤔

Ifølge aksptansekriteriene:

GIVEN a call from a service owner that creates or updates a activity log entry
WHEN the performedBy field is supplied
AND the performedBy value is longer than 100 bytes
THEN the request should fail with an 400 error